demo-config.json

```diff
@@ -16,6 +16,10 @@
           "Magento-Environment-Id": "f38a0de0-764b-41fa-bd2c-5bc2f3c7b39a"
         }
       },
+      "company-switcher": {
+        "company-header-key": "X-Adobe-Company",
+        "company-session-key": "DROPIN__COMPANYSWITCHER__COMPANY__CONTEXT"
+      },
```

Comment:
What are these for? It's probably fine to define them as constants within the dropin, unless you think they can/should be customized by a merchant.

Reply:
Well, we talked about this on one of the meetings and @fnhipster mentioned that there are theoretical edge cases with mesh that might reauire header name to be configurable, so this what are these for. 
I kept session key configurable too just for consistency.

Reply:
How is this header defined in the backend? Is it possible for a customer to change it?

Carlos did initial dive into business logic so I don't have any of that context.

Reply:
@sirugh I believe the use case that was described is when we have API Mesh in between Storefront and Adobe Commerce backend where user could change the header signature.

Reply:
I don't have any info on this, unfortunately. My assumption is that it is an edge case for some very specific mesh configurations that I don't have real world examples for.

Reply:
Unless we anticipate a custom header and session key being asked for by real customers, I'd prefer to avoid additional complexity. Making it dynamic means we have to document and maintain that interaction, and the header/session key becomes part of the versioned "API" for the dropin.

Reply:
And we are back where we started

Reply:
Regarding this configuration thread - it is about how a customer might override the header or session keys. For that, they already have full control over "integration" code once they fork/clone the boilerplate.

Carlos is saying - change the variable to hardcode to some constant, and then document how this can be updated if necessary.

It does not need to be something in public config.